sktime/forecasting/model_selection/_split.py

```diff
@@ -666,8 +633,7 @@ def _get_train_window(
             return np.argwhere(
                 (y >= y[max(train_start, 0)]) & (y <= y[min(split_point, len(y)) - 1])
             ).flatten()
-        else:
-            return np.array([], dtype=int)
+        return np.array([], dtype=int)
```

Comment:
same here, why?

Reply:
same as in the other comment (https://github.com/sktime/sktime/pull/4108#discussion_r1070364781)